docs/Modding/guides/tools/rpakmodding.md

```diff
@@ -218,6 +258,9 @@ Example: ``"common.rpak": "my_new.rpak"``
 
 #### Postload
 
+!!! warning
+    Postload is now deprecated. New RPak mods should use dynamic map loading to load their RPaks.
+
```

Comment:
Due to https://github.com/R2Northstar/NorthstarLauncher/issues/821 we probably shouldn't deprecate Postload. In fact, it should probably be moved out of the "old" section